src/arena/ArenaStats.tsx

```diff
@@ -0,0 +1,86 @@
+import { I18n } from "@aws-amplify/core";
+
+import StatsBlock from "../components/StatsBlock";
+import { GenMission } from "./interface";
+
+interface ActivePersonMission {
+  dailyCompletion: number;
+  dailyScore: number;
+  missions: GenMission[];
+}
+
+interface ActivePersonReview {
+  code: string;
+  content: string;
+  name: string;
+}
+
+interface Props {
```

Comment:
Nit: rename to `ArenaStatsProps` - in case you are wondering why I am requesting these name changes, it is because at some point in the future - we might put them in a centralized file/npm package that we auto-generate and consume by `exporting` them - and so having them all called `Props` is not ideal as we may rename them.